pkg/network/ebpf/c/protocols/http.h

```diff
@@ -2,7 +2,7 @@
 #define __HTTP_H
 
 #include "bpf_builtins.h"
-#include "tracer.h"
+#include "../tracer.h"
```

Comment:
`pkg/network/ebpf/c` should be in the include path, so this should not be needed?

Reply:
![image](https://user-images.githubusercontent.com/17148247/198837574-c68d9dff-4b4e-41ac-ac1f-f30dfcafa50b.png)

Reply:
https://github.com/DataDog/datadog-agent/pull/14089#discussion_r1008710705

Reply:
:+1: